### DISCUSSION TITLE
Problem installing the addon 'r.vector.ruggedness'

### DISCUSSION THREAD

**Discussion by josediazrmos** [OSGeo/grass#2191] [Q&A] [answered]
Hi everyone! 

I am new here and working with GRASS too. Normally I work with QGIS, but when I tried to use the VRM addon from the GRASS extension I found that it is not available. So I downloaded GRASS but discovered that you have to install the addon that I need. 

So I went to Settings > Addons extensions > Install extension from addons and selected r.vector.ruggedness. Nevertheless, it appears the following message: "ERROR: Cannot open URL: http://wingrass.fsv.cvut.cz/grass80/addons/grass-8.0.0dev/r.vector.ruggedness.zip". Do you have any idea of how can I fix this? 

Thanks!!

**Comment by veroandreo** [comment, marked as answer]
This seems the same problem reported in this [thread](https://lists.osgeo.org/pipermail/grass-user/2022-February/082861.html). Please, re-install GRASS using the installer here: https://grass.osgeo.org/grass80/binary/mswindows/native/WinGRASS-8.0.0-2-Setup.exe that has been fixed.

The link to the installer is also fixed now in the GRASS website, it should appear after midnight or so, in the daily builds.

**Reply by neteler** [reply]
> The link to the installer is also fixed now in the GRASS website, it should appear after midnight or so, in the daily builds.

(I have enforced the website update, should be now fine.)

**Reply by josediazrmos** [reply]
Everything works fine now. Thanks!!
